inputfiles/knownTypes.json

```diff
@@ -81,7 +81,8 @@
         "RsaPssParams",
         "Transferable",
         "VideoFacingModeEnum",
-        "MouseWheelEvent"
+        "MouseWheelEvent",
+        "WorkerOptions"
```

Comment:
This and line 114 are not needed if you use `type` instead of `override-type`.

Reply:
@saschanaz I changed the `addedTypes` to use `type`, but if I remove the additions to `knownTypes` and exec `npm run build && npm run baseline-accept`, the `WorkerOptions` interface is being removed. What am I missing?